nf_core/pipeline-template/README.md

```diff
@@ -40,11 +40,13 @@ On release, automated continuous integration tests run the pipeline on a full-si
 3. Download the pipeline and test it on a minimal dataset with a single command:
 
     ```console
-    nextflow run {{ name }} -profile test,<docker/singularity/podman/shifter/charliecloud/conda/institute>
+    nextflow run {{ name }} -profile test,YOURPROFILE
     ```
+    Note that some form of configuration will be needed so that Nextflow knows how to fetch the required software. This is usually done in the form of a config profile (`YOURPROFILE` in the example command above). You can chain multiple config profiles in a comma-separated string.
```

Comment:
```suggestion

    Note that some form of configuration will be needed so that Nextflow knows how to fetch the required software. This is usually done in the form of a config profile (`YOURPROFILE` in the example command above). You can chain multiple config profiles in a comma-separated string.
```